src/controllers/dynakube/dynakube_controller_test.go

```diff
@@ -342,3 +384,154 @@ func createFakeClientAndReconcile(mockClient dtclient.Client, instance *dynatrac
 
 	return r
 }
+
+// generateStatefulSet prepares an ActiveGate StatefulSet after a Reconciliation of the Dynakube with a specific feature enabled
+func generateStatefulSet(name, namespace, feature, kubeSystemUUID string) *appsv1.StatefulSet {
```

Comment:
This name could be confused with actual (not test) code, so naming like `generateTestingStatefulSet` or something similar